driver/manifest/manifest.go

```diff
@@ -211,13 +180,58 @@ func Load(path string) (*Manifest, error) {
 	if err != nil {
 		return m, err
 	}
+	if err := LoadBuildManifest(m, open); err != nil {
+		return m, err
+	}
 	m.Maintainers, err = Maintainers(open)
 	if err != nil {
 		return m, err
 	}
 	return m, nil
 }
 
+func extractVersion(s string) string {
```

Comment:
Please document what this is meant to do, e.g.,

```go
// extractVersion parses a semantic version from a Docker image name having
// the form base:label, where label is either a version or version-tag pair.
```

At least I think that's what's going on here. Maybe the name should indicate that too, say maybe `extractImageVersion`?